js/stateless.js/src/instructions/compressed-pda.ts

```diff
@@ -0,0 +1,138 @@
+import { Program, AnchorProvider, setProvider } from "@coral-xyz/anchor";
+import {
+  PublicKey,
+  TransactionInstruction,
+  Keypair,
+  Connection,
+} from "@solana/web3.js";
+import { IDL, PspCompressedPda } from "../idls/psp_compressed_pda";
+import { confirmConfig } from "../constants";
+import { useWallet } from "../wallet";
+import { UtxoWithMerkleProof } from "../state";
+
+export type CompressedTransferParams = {
+  /** Utxos with lamports to spend as transaction inputs */
+  fromBalance: UtxoWithMerkleProof[];
+  /** Solana Account that will receive transferred compressed lamports as utxo  */
+  toPubkey: PublicKey;
+  /** Amount of compressed lamports to transfer */
+  lamports: number | bigint;
+};
+
+/**
+ * Create compressed account system transaction params
+ */
+export type CreateCompressedAccountParams = {
+  /**
+   * Optional utxos with lamports to spend as transaction inputs.
+   * Not required unless 'lamports' are specified, as Light doesn't
+   * enforce rent on the protocol level.
+   * */
+  fromBalance: UtxoWithMerkleProof[];
+  /** Public key of the created account */
+  newAccountPubkey: PublicKey;
+  /** Amount of lamports to transfer to the created compressed account */
+  lamports: number | bigint;
+  /** Public key of the program to assign as the owner of the created compressed account */
+  programId: PublicKey;
+};
+
+/**
+ * Example usage:
+ * ```typescript
+ * const tx = await LightSystemProgram.transfer({
+ *  fromBalance: [
+ *   {
+ *      owner: new PublicKey("..."),
+ *      data: [],
+ *      lamports: 2000000000n,
+ *      leafIndex: 0n,
+ *      hash: 0n,
+ *      merkletreeId: 0n,
+ *      merkleProof: [],
+ *   },
+ * ],
+ * toPubkey: new PublicKey("..."),
+ * lamports: 1000000000n,
+ * });
+ * ```
+ */
+export class LightSystemProgram {
```

Comment:
yup. idea here is to simplify interacting with it since we know the idl and ixs and all and therfore can use it as a class singleton (and for example abstract away the anchor syntax with a thin wrapper)